pkg/util/k8sutil.go

```diff
@@ -121,3 +123,27 @@ func IsPodFaulty(pod *corev1.Pod) (bool, error) {
 	}
 	return false, nil
 }
+
+func NewEvent(name string, p *v1alpha1.PravegaCluster, reason string, message string, eventType string) *corev1.Event {
+	now := metav1.Now()
+	operatorName, _ := k8s.GetOperatorName()
+	event := corev1.Event{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
```

Comment:
`Name:` seems wrong; `GenerateName` is probably more appropriate.   Please do a little more research on this.